bin/build-debug.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $npm_package_browserify_options ]; then
+  options=$npm_package_browserify_options
+else
+  options=$NPM_PACKAGE_BROWSERIFY_OPTIONS
```

Comment:
Sure, I like that. For whatever reason they get capitalized when going through CMD and bash is case sensitive. Pretty evil.